### ISSUE TITLE
Edit with Elementor Button Not Showing when Autoptimize Activated

### ISSUE THREAD

**Issue by oliveandlake** [elementor/elementor#12995]
## Prerequisites

- [ x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [ x] The issue still exists against the latest stable version of Elementor.

## Description

Exactly as it says in the title. 

Here's a video

[https://www.loom.com/share/5a2265ed3cda41e393225bb1db92a79a](https://www.loom.com/share/5a2265ed3cda41e393225bb1db92a79a)

## Steps to reproduce
1. Activate Autoptimize
2. Go to Front end.
3. See no "Edit with Elementor" button!!

## Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x ] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by makaus** [comment]
Turn off "Also optimize for logged in editors/ administrators?"

**Comment by oliveandlake** [comment]
Aha, thanks @makaus - that's worked. Actually, I should have it off anyway. Best not to have "optimised" pages for logged in users.

**Comment by shilo-ey** [comment]
I merged this issue with  #12951 , which is on the same subject so we could deal with it more efficiently. 
Please feel free to continue the conversation there.

Thanks for your feedback!
